.github/renovate.json5

```diff
@@ -50,6 +50,7 @@
         '(:|\\s+)earthly/earthly:(?<currentValue>.+?)[\\s\\n/]+',
         'the `latest` tag is `(?<currentValue>.+?)`',
         'earthly/compare/(?<currentValue>.+?)\\.\\.\\.main',
+        'earthly/satellite:(?<currentValue>.+?)',
```

Comment:
@Ido did I set this up correctly? The intent is to update lines that look like `earthly/satellite:v0.8.2`